TheHive/thehive/create_alert.py

```diff
@@ -94,17 +94,19 @@ def run(self, arguments: dict[str, Any]):
                             )
                         )
 
+        alert_type = f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}"
```

Comment:
Maybe try to cut it in a way that will be less strange in the UI ?

```suggestion
        alert_type = f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}"
        if len(alert_type) > 32:
            alert_type = arg_alert['alert_type']['category'][:32]  # limit to 32 char, max of thehive api
```